peer_mentoring/chat/templates/chat/room.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.html %}
```

Comment:
This template was not findable because of where the templates directory is located:

I moved it out of here:

```
(venv) √ peer_mentoring (updates) $ pwd
/Users/bbelderbos/code/PDM/projects/jamesd/peer_mentoring/peer_mentoring/peer_mentoring
(venv) √ peer_mentoring (updates) $ ls
__init__.py	__pycache__	asgi.py		settings.py	urls.py		wsgi.py
```

To one level higher, so now I have it here:

```
(venv) √ peer_mentoring (updates) $ cd ..
(venv) √ peer_mentoring (updates) $ pwd
/Users/bbelderbos/code/PDM/projects/jamesd/peer_mentoring/peer_mentoring
(venv) √ peer_mentoring (updates) $ ls
account_management	db.sqlite3		interests		peer_mentoring		requirements.txt
chat			groups			manage.py		posts			templates
```

That matches what you have in settings.py:

```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [os.path.join(BASE_DIR, 'templates'), os.path.join(BASE_DIR, 'templates/allauth')],
```

(I printed `BASE_DIR` to discover this issue)

With the directory moved to one level higher the base.html is found and the http://localhost:8000/ yields the child template.

HTH
Bob